src/btree/bt_cursor.c

```diff
@@ -1881,12 +1882,17 @@ __cursor_truncate(WT_CURSOR_BTREE *start, WT_CURSOR_BTREE *stop,
 
     for (;;) {
```

Comment:
There is a `retry` label just above, does it mean we need to restart the counter as well? When it retries, does it cancel what the cursor has truncated? I had a look at the code and it does not seem to cancel any operation but I would be happy if you could double check.

Reply:
I don't think we should restart the counter because it seems like the cursor would just retry from the position it's currently at. I can definitely look into this further though.

Reply:
That was my understanding too and the code does not seem to retry the operation as in, retry from the beginning. Resolving the comment, thanks!